sympy/sets/sets.py

```diff
@@ -908,6 +908,9 @@ def _union(self, other):
 
         See Set._union for docstring
         """
+        # For unions of intervals UniversalSet is S.Reals
+        if other.is_UniversalSet or other == S.Reals:
+            return S.Reals
```

Comment:
On the issue page, Matt Rocklin said, "Easy option is to add a global rule in reduce that UniversalSet + anything -> UniversalSet" so instead of return S.Reals, can you just return S.UniversalSet? And then maybe the tests will not fail.